Yoast/wpseo-woocommerce
# Remove link count columns (from Yoast SEO)

## Summary

This PR can be summarized in the following changelog entry:
* Remove the Yoast SEO link count columns from the product page when the setting `Remove Yoast SEO columns` is checked.

## Test instructions

This PR can be tested by following these steps:

* Checkout `trunk`
* Go to the WooCommerce SEO settings and check the setting `Remove Yoast SEO columns`
* Go to the product overview and see all Yoast SEO columns except (the link count columns) being removed.
* Checkout this branch and see that the columns are removed

Fixes #162
